pyvista/utilities/common.py

```diff
@@ -23,9 +23,11 @@ def _coerce_pointslike_arg(
 
     Returns
     -------
-    np.ndarray
+    points : np.ndarray
```

Comment:
I don't know how consistent the codebase is in this, but there's precedent of keeping only types here even if there are multiple return values, e.g. https://github.com/pyvista/pyvista/blob/709293784756e7f0fb4f4125a95fafc02eab0da5/pyvista/core/filters/poly_data.py#L3181-L3189

Reply:
That's the precedent, but we can change this if we feel that it needs to be changed. numpydoc specifies that this is fine.

Personally I like how clean it looks without the variable return name, and it also makes maintenance a bit easier.

Reply:
Three non-conforming cases are probably all committed by me.  Sorry.  It is my personal preference, but I should be more careful with the pyvista style.